pkg/util/push/push.go

```diff
@@ -115,7 +115,7 @@ func handler(maxRecvMsgSize int,
 			return &req.WriteRequest, cleanup, nil
 		}
 		req := newRequest(supplier)
-		if _, err := push(ctx, req); err != nil {
+		if err := push(ctx, req); err != nil {
```

Comment:
what if `err` is nil? :) looks strange we don't handle that *at all*.

Reply:
Sounds like it has always been like this? I guess we rely on default golang HTTP handling.

Reply:
I confirm that it was already like that. In this case "no error" is implicitly translated into an HTTP 200.
I have added a test to show that.

Reply:
I agree that this PR is not changing. I've just never noticed it before :)